src/main/resources/templates/db_scripts/schema.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
I'm guessing this is needed and not superfluous.  Is it part of the liquidbase work?

Reply:
yes